### PR TITLE
[CenterOfMass] Added support to enter a mass and other improvements

### PR DESCRIPTION
**Pull request by s-quirin**
- [x] Please check this box if you're not submitting a new macro.   

---
### Changelog:
- **New**: Masses are editable (a highly requested feature)
- **New**: Highlights the solid you are working on
- **New**: No duplicate entries when container and content selected concurrently
- **New**: Pie charts show density relation in combo box
- **New**: Legibility of text on colored combo box with WCAG21 1.4.6 Contrast (Enhanced) conformity
- **Fix**: A Part used as a container for meshes (e.g. .stl's) is recognized correctly
- **Fix**: Fixes for error and language handling, material editing, combo box and GUI size adjustment

### CODE REVIEW

File: Information/CenterOfMass.FCMacro [hunk 1]
```diff
@@ -97,6 +105,7 @@ class CenterofmassDock(QtGui.QDockWidget):
     def __init__(self):
         super().__init__()
         self.setAttribute(QtCore.Qt.WA_DeleteOnClose, True)    # free memory
+        self.setLocale(QtCore.QLocale.English)
```
Comment (galou) [inline Information/CenterOfMass.FCMacro:108]: Can you please explain why you need to set the locale to English?

File: Information/CenterOfMass.FCMacro [hunk 2]
```diff
@@ -25,8 +25,8 @@
 __Name__ = 'CenterOfMass'
 __Comment__ = 'Compute and show the center of mass for multiple solids'
 __Author__ = 'chupins, s-quirin'
-__Version__ = '0.5.8'
-__Date__ = '2022-05-04'
+__Version__ = '0.6'
```
Comment (galou) [inline Information/CenterOfMass.FCMacro:28]: Please add the patch version next time. I'll do it this time myself.

**Review by galou — APPROVED** [review]

**Comment by galou** [comment]
Thanks for your contribution!

File: Information/CenterOfMass.FCMacro (hunk 1 above)
Comment (s-quirin) [reply Information/CenterOfMass.FCMacro:108]: The user interface texts are in English only. The setting ensures uniform decimal separator for line edit and spin box fields.

**Pull request merged**